src/Raven.Server/Documents/ETL/EtlLoader.cs

```diff
@@ -229,6 +240,15 @@ private void HandleChangesSubscriptions()
                         else
                             connectionStringNotFound = true;
                         break;
+                    case EtlType.ElasticSearch:
+                        elasticsearchConfig = config as ElasticSearchEtlConfiguration;
+                        if (_databaseRecord.ElasticSearchConnectionStrings.TryGetValue(config.ConnectionStringName, out var elasticsearchConnection))
```

Comment:
Minor: elasticsearchConnection ->elasticSearchConnection. In general for consistency let's do the case sensitive search for "elasticsearch" / "Elasticsearch" and rename to "elasticSearch" / "ElasticSearch"

Reply:
done